lib/puppet_x/puppetlabs/iis/property/path.rb

```diff
@@ -0,0 +1,17 @@
+# (c:\directory or c:/directory) or \\server\share
```

Comment:
Move the comment closer to the regex, or remove from top of file

Reply:
Deleted.